pkg/scheduler/eventhandlers.go

```diff
@@ -430,7 +430,7 @@ func addAllEventHandlers(
 		)
 	}
 
-	// On add and delete of PVs, it will affect equivalence cache items
+	// On add and update of PVs, it will affect equivalence cache items
```

Comment:
Equivalence cache has been removed, so could you help remove those wordings?

```suggestion
	// On add and update of PVs.
```

Reply:
ok, has removed the note.

Reply:
Yes, please update the comment to be "// On add and update of PVs."

Reply:
And please squash the commits along with the new edit.